packages/frontend-api/src/Voter/CustomerUserVoter.php

```diff
@@ -71,7 +74,12 @@ protected function isUnauthenticatedAccessAllowed(array $inputData, TokenInterfa
      */
     protected function isRoleApiAllGranted(array $inputData, TokenInterface $token): bool
```

Comment:
You got it right. The method naming approach I've adopted in Voter(s) is based on the specific role being checked, rather than the logic of the check itself. I believe this approach enhances readability and maintainability, especially when overriding methods. It allows us to quickly identify what role a method is associated with, without delving into the specifics of the checking logic.

I am open to discussion